### ISSUE TITLE
How do you mount a drive?

### ISSUE THREAD

**Issue by 8465231** [maharmstone/btrfs#391]
I can not find any instructions on how to use this program at all? It just says how to install it but nothing about how to mount a drive?

Is there a GUI for it? I saw this: "The DLL file shellbtrfs.dll provides the GUI interface,"

How would I access the GUI?

I have some btrfs drives that were formatted in ubuntu but naturally windows does not detect any file system on them.

It seems like I am missing a help / instruction write-up file somewhere?

**Comment by TheMadHau5** [comment]
Please set the partition type as MS Basic Data partition, as all other partitions (such as linux partitions) are ignored by the operating system. Setting this should not affect linux though.

**Comment by maharmstone** [comment]
@TheMadHau5 - this is incorrect.

@8465231 - have you installed it directly, or via Chocolatey? Some people have had similar issues with Chocolatey in the past.

**Comment by 8465231** [comment]
> @8465231 - have you installed it directly, or via Chocolatey? Some people have had similar issues with Chocolatey in the past.

It is installed directly with the ini as per the readme.

So there is no way to manually interact with the program at all? Odd I just kinda figured there would be an interface of some sort to say, mount drive x to letter y etc.

**Comment by maharmstone** [comment]
That's not how Windows works - Windows automounts whatever it can. There's no `mount` command or what have you.

Is the driver actually running? (Try `sc query btrfs`.)

**Comment by 8465231** [comment]
Yeah but since windows can't detect the btrfs file system seems like you would need a way of mounting it. I was thinking along the lines of how you mount an ISO or something but guess I was thinking wrong.

Ran that command: 

SERVICE_NAME: btrfs
        TYPE               : 1  KERNEL_DRIVER
        STATE              : 4  RUNNING
                                (STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)
        WIN32_EXIT_CODE    : 0  (0x0)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0

I have plugged in 7 or 8 BTRFS drives that work fine on ubuntu but none of them are mounted on windows. If I look at the disk management it sees the disks but no valid file systems.

**Comment by 8465231** [comment]
After some searching I found this: https://github.com/maharmstone/btrfs/issues/388

Tried enabling automount. This seems to of fixed the disks not mounting issue. I can now mount the disks!

How does mounting raid disks work? can I just plug them in one at a time and it will automount when the last one is detected? Or so I have to have them all inserted and boot that way?

Gonna move over some raided disks in the coming days and want to know what to expect.
